spec/bolt_ext/server_spec.rb

```diff
@@ -25,22 +25,22 @@ def app
 TASK
 
       body = {
-        'task' => {
-          'name' => 'echo',
-          'metadata' => {
-            'description' => 'Echo a message',
-            'parameters' => { 'message' => 'Hello world!' }
+        'task': {
```

Comment:
That seems weird, why do the keys need to be symbols?

Reply:
They don't, it was just an aesthetic change.

Reply:
I think the travis failures were transient? My last commit just added a print statement to figure out why it was passing locally but not in travis, and then travis started passing :woman_shrugging: